client/src/project/Project.present.js

```diff
@@ -263,6 +263,58 @@ function ProjectViewHeaderOverviewDescription({ settingsReadOnly, description, s
   return null;
 }
 
+const ProjectSuggestActions = (props) => {
+  useEffect(() => {
+    props.fetchProjectSuggestions();
+  }, []); // eslint-disable-line
+
+  const datasets = props.core?.datasets;
+  const cantCommitsReadme = props.readme?.cantCommits;
+  const hasDatasets = typeof (datasets) === "object" ? datasets.length > 0 : true;
+
+  if (cantCommitsReadme > 1 && hasDatasets) return "";
+
+  const gitlabIDEUrl = props.externalUrl !== "" && props.externalUrl.includes("/gitlab/") ?
+    props.externalUrl.replace("/gitlab/", "/gitlab/-/ide/project/") : null;
+  const addReadmeUrl = `${gitlabIDEUrl}/edit/${props.core.default_branch}/-/README.md`;
+
+  const suggestionReadme = cantCommitsReadme > 1 ? null
+    : <li><p style={{ fontSize: "14px" }}>
+      <a className="mx-1" href={addReadmeUrl} target="_blank" rel="noopener noreferrer">
+        <strong>Add README.md</strong>
```

Comment:
The wording here is exactly as in the design, but after actually using the feature, I want to make a change:

```suggestion
        <strong>Edit README.md</strong>
```